lib/tasks/auth_tokens.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+namespace :auth_tokens do
+  desc "Verify each authorized AuthToken that it's still authorized"
+  task :reverify_authorized, %i[count start] => :environment do |_task, args|
+    exit if ENV["READ_ONLY"].present?
+    args.with_defaults(count: 500, start: nil)
+    last_id = "none"
+
+    begin
+      AuthToken
+        .authorized
```

Comment:
WDYT about sorting this list in either one of two ways:

* alphabetically by `id` so that, if we have to reprocess the list, the `last_id` in the error message will allow us to restart from there if desired
* oldest-to-newest by `updated_at` so that the oldest `AuthTokens` are processed first

Reply:
Sounds good to me!